domains/Discussions/Models/Question.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Domains\Discussions\Models;
+
+use Domains\Accounts\Models\User;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Question extends Model
+{
+    use SoftDeletes;
+
+    public function author(): BelongsTo
+    {
+        return $this->belongsTo(User::class);
```

Comment:
We should take into consideration that a User resource can be soft-deleted. However, we might not want to delete resources that that User created. If we don't define this relationship with the `withTrashed()` scope, we won't have access to the soft-deleted User information.

I think that, by default, we should load all relationship information.

Reply:
makes sense !